src/SASjs.ts

```diff
@@ -107,6 +107,34 @@ export default class SASjs {
     return await this.sasViyaApiClient!.getExecutableContexts(accessToken)
   }
 
+  public async createContext(
+    contextName: string,
+    launchContextName: string,
+    sharedAccountId: string,
+    autoExecLines: string[],
+    authorizedUsers: string[],
+    accessToken: string
+  ) {
+    if (this.sasjsConfig.serverType !== ServerType.SASViya) {
+      throw new Error('This operation is only supported on SAS Viya servers.')
+    }
+    return await this.sasViyaApiClient!.createContext(
+      contextName,
+      launchContextName,
+      sharedAccountId,
+      autoExecLines,
+      authorizedUsers,
+      accessToken
+    )
+  }
+
+  public async deleteContext(contextId: string, accessToken?: string) {
```

Comment:
all public functions need to be documented

Reply:
I've added comments for the new functions I added - but if you look closely, none of the other public functions in this file are documented either.
We'll need to address that as part of a separate issue.

Reply:
Yes, you are right that the majority of the functions are not documented and we should create a separate issue. But still, it'll be easier to add comments as we go.